common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp

```diff
@@ -490,6 +491,18 @@ inline geometry_msgs::msg::Point transformPoint(
   return transformed_point;
 }
 
+inline geometry_msgs::msg::Point32 transformPoint(
+  const geometry_msgs::msg::Point32 & point32, const geometry_msgs::msg::Pose & pose)
+{
+  const auto point =
+    geometry_msgs::build<geometry_msgs::msg::Point>().x(point32.x).y(point32.y).z(point32.z);
+  const auto transformed_point = tier4_autoware_utils::transformPoint(point, pose);
+  return geometry_msgs::build<geometry_msgs::msg::Point32>()
+    .x(transformed_point.x)
+    .y(transformed_point.y)
+    .z(transformed_point.z);
+}
```

Comment:
@kenji-miyake 
Thank you for your advice! I made code simple in https://github.com/autowarefoundation/autoware.universe/pull/1981/commits/736e13f9e167b6135b58cc0b301ba277ee502c6a